content/courses/mobile/intro-to-solana-mobile.md

```diff
@@ -820,17 +823,21 @@ export type ProgramProviderProps = {
   children: ReactNode;
 };
 
-export function ProgramProvider(props: ProgramProviderProps) {
-  const { children } = props;
+export function ProgramProvider({ children }: ProgramProviderProps) {
   const { connection } = useConnection();
   const [program, setProgram] = useState<Program<AnchorCounter> | null>(null);
   const [counterAddress, setCounterAddress] = useState<PublicKey | null>(null);
 
   const setup = useCallback(async () => {
     const programId = new PublicKey(
-      "ALeaCzuJpZpoCgTxMjJbNjREVqSwuvYFRZUfc151AKHU",
+      "ALeaCzuJpZpoCgTxMjJbNjREVqSwuvYFRZUfc151AKHU", //public key, do not expose anything else
```

Comment:
```suggestion
      "ALeaCzuJpZpoCgTxMjJbNjREVqSwuvYFRZUfc151AKHU",
```

Honestly people should know not to expose secret keys by this pointm the code already says publickey, and secret keys are usually in array of numbers format.